src/Commands/MakeCommand.php

```diff
@@ -60,6 +63,10 @@ public function handle()
                 $test && $this->line("<options=bold;fg=green>TEST:</>  {$this->parser->relativeTestPath()}");
             }
 
+            if ($pest) {
```

Comment:
I think `if ($pest) {` is redundant/unneeded here, we already have `$pest &&` below it.